crates/sui-graphql-rpc/src/types/transaction_block.rs

```diff
@@ -244,188 +61,27 @@ pub(crate) struct TransactionBlockFilter {
     pub transaction_ids: Option<Vec<String>>,
 }
 
-impl BalanceChange {
-    fn from(balance_changes: Vec<Option<Vec<u8>>>) -> Result<Vec<Option<BalanceChange>>, Error> {
-        let mut output = vec![];
-        for balance_change_bcs in balance_changes.into_iter().flatten() {
-            let balance_change: NativeBalanceChange = bcs::from_bytes(&balance_change_bcs)
-                .map_err(|_| {
-                    Error::Internal("Cannot convert bcs bytes to BalanceChange".to_string())
-                })?;
-            let balance_change_owner_address =
-                balance_change.owner.get_owner_address().map_err(|_| {
-                    Error::Internal("Cannot get the balance change owner's address".to_string())
-                })?;
-
-            let address =
-                SuiAddress::from_bytes(balance_change_owner_address.to_vec()).map_err(|_| {
-                    Error::Internal(
-                        "Cannot get a SuiAddress from the balance change owner address".to_string(),
-                    )
-                })?;
-            let owner = Owner { address };
-            let amount =
-                BigInt::from_str(balance_change.amount.to_string().as_str()).map_err(|_| {
-                    Error::Internal(
-                        "Cannot convert balance change amount to BigInt amount".to_string(),
-                    )
-                })?;
-            output.push(Some(BalanceChange {
-                owner: Some(owner),
-                amount: Some(amount),
-                coin_type: Some(MoveType::new(
-                    balance_change.coin_type.to_canonical_string(true),
-                )),
-            }))
-        }
-        Ok(output)
+#[ComplexObject]
+impl TransactionBlock {
+    /// A 32-byte hash that uniquely identifies the transaction block contents, encoded in Base58.
+    /// This serves as a unique id for the block on chain
```

Comment:
nit: add a dot at the end of the sentence.

Reply:
Let me make this change in a follow-up PR -- this is intentionally a pure code motion change, to isolate that from other, substantive changes.